cmd/build/command.go

```diff
@@ -103,7 +103,12 @@ func Build(ctx context.Context) *cobra.Command {
 				}
 			}
 
-			return builder.Build(ctx, options)
+			err = builder.Build(ctx, options)
+			if err != nil {
+				return suggest.UserFriendlyError(err)
```

Comment:
I find it odd to transform it here. There are many things that can go wrong in the middle to transform the error here. Maybe we should transform it on the serialization? and check if it's a userError like we do in other parts of the code like [preview list](https://github.com/okteto/okteto/blob/master/cmd/preview/list.go#L103)

Reply:
this is a good topic to discuss, let me explain my idea: the function `suggest.UserFriendlyError` is responsible for improving all errors (not only manifest), in the near future we might need to improve error messages related to another aspect of the CLI.

I find very intruisive in the code to change literally every single file to wrap error return with a function specific for that error.

This approach is more generic and wrapping the whole CLI instead. I am sure it comes with a downside but from the positive side we have a single point of entrance for this logic.